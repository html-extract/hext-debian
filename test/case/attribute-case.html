<div>
  <span class="lower case attribute"></span>
  <span cLaSs="mixed case attribute"></span>
  <span CLASS="upper case attribute"></span>
</div>
