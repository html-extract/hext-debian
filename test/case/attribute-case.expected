{"class":"lower case attribute"}
{"class":"mixed case attribute"}
{"class":"upper case attribute"}
